api/models.py

```diff
@@ -1,19 +1,15 @@
 import datetime
 from sqlalchemy import func
 from sqlalchemy import event
-from .config import app
+from sqlalchemy.orm import Session
 
-from .config import db
+from .config import app, db
 
 
 def orm_log_after(tablename, method, owner_id, id):
     app.logger.debug(f"user_id {owner_id} finished {method} a {tablename} {id}")
 
 
-def orm_log_before(tablename, method, owner_id):
-    app.logger.debug(f"user_id {owner_id} started {method} a {tablename}")
-
-
 track_singers = db.Table(
```

Comment:
make it rather a model - this way you can inherit it from base model and have update_time and create_time